pages/ybribe/index.tsx

```diff
@@ -50,7 +51,7 @@ function	GaugeList(): ReactElement {
 		}
 		return gauges.filter((gauge): boolean => {
 			const hasCurrentRewardsV3 = currentRewards?.v3?.[toAddress(gauge.gauge)] !== undefined;
```

Comment:
I'm wondering if `currentRewards` has the same new structure, @xgambitox do you know?